packages/icons/scripts/generateIcons.mjs

```diff
@@ -310,6 +310,7 @@ const cssOutputPath = path.join(__dirname, "../saltIcons.css");
 const fileArg = process.argv.splice(2).join("|");
 const templatePath = path.join(__dirname, "./templateIcon.mustache");
 const allPath = path.join(basePath, "../stories/icon.all.ts");
+const allCorePath = path.join(basePath, "../../core/stories/badge/icon.all.ts");
```

Comment:
Is this extra file really needed, or can be derived from existing ones?

Reply:
removed